entrypoint.sh

```diff
@@ -270,6 +291,11 @@ test_go_ceph() {
     fi
     export CEPH_CONF
 
+    if [[ ${MIRROR_CONF} ]]; then
+      setup_mirroring
+      export MIRROR_CONF
+    fi
+
```

Comment:
Unless I'm reading the code wrong, moving the mirroring setup to this part of the file will cause issues for actions like `make check-implements` amongst other things. Was there a reason for moving this and not just removing the "not nautilus" part of the if statement?
Basically I want to this to happen after the $TEST_RUN == NONE check, so that we don't do a lot of potentially slow stuff before we decide not to execute the integration tests.

You could claim that the script is kinda messy and a tad overloaded when it comes to this stuff, I'd agree but I'd suggest that'd be for a follow up PR to refactor entrypoint.sh. I would like to keep this one small. :-)

Reply:
Before I made the above change, setting `ENTRYPOINT_ARGS="--test-run=NONE --pause` would result in the two clusters that got spun up not being automatically mirrored. However, if fixing that one problem causes other, bigger problems, happy to revert it. But I'm not sure I understand why it would break `make check-implements` (although I do see why it would slow it down).

Reply:
Yes. I changed "break" to "cause issues" which was my way of saying "slow". :-)
IIRC it could be a significant slow down too. 

I forgot that implements has it's own pseudo-target. Sometimes I try to get thru all my emails/feedback too fast without checking everything. Still, it might be an issue I just haven't checked all the code paths. 

Regardless, I'm trying to understand why this might fail in the previous position. I checked the makefile and the test-containers-test_ceph_{a,b} rules do use --test-run=NONE --pause as arguments. I think this means that if the old position was outright incorrect our CI would always fail. So I'm not sure why it might be failing in your case.

I can only suggest moving it back to the previous position for now, so that if it does fail  it fails in the CI so that we all can see it.

Reply:
Just to clarify: running with `--test=NONE --pause` wasn't failing for me, it was just not setting up mirroring (as expected, though inconvenient for my dev loop), because `setup_mirroring` was called after the `$TEST_RUN == NONE` check, so naturally it never got executed.

Reply:
Got it, thanks. I appreciate changing it back as I think it would interfere with my workflow. :-)